scheduler/models.py

```diff
@@ -1,20 +1,22 @@
 from __future__ import unicode_literals
+
 import importlib
 from datetime import timedelta
 
 import croniter
-
+import django_rq
 from django.conf import settings
 from django.core.exceptions import ValidationError
 from django.db import models
 from django.templatetags.tz import utc
 from django.utils.encoding import python_2_unicode_compatible
+from django.utils.timezone import now
 from django.utils.translation import ugettext_lazy as _
-
-import django_rq
 from model_utils import Choices
 from model_utils.models import TimeStampedModel
 
+RQ_SCHEDULER_INTERVAL = getattr(settings, "DJANGO_RQ_SCHEDULER_INTERVAL", 60)
+
```

Comment:
This is currently undocumented in the README. I've not done enough testing on the ramifications of changing this from the default `interval` of 60 set in Django RQ's `get_scheduler` function.
I have written checks that prevent saving jobs where the given `job_interval` either shorter than this interval or not a multiple of it (seemed the safest / most restrictive approach). I felt this was necessary as part of adding seconds as an `interval_unit`.